lips/lip-1.mdx

```diff
@@ -302,16 +319,17 @@ For a travel rule data exchange, the [command_type](basic_building_blocks.md#com
 | _ObjectType   | str  | Y             | The fixed string `PaymentCommand` |
 | payment| [`PaymentObject`](#paymentobject) | Y | contains a `PaymentObject` that either creates a new payment or updates an existing payment. Note that strict validity check apply when updating payments, that are listed in the section below describing these Objects. An invalid update or initial payment Object results in a Command error
 | _reads | JSON Object map |  Y | Must be an Object containing mappings of `reference_id` to latest version as represented by the `cid` of the latest Command which successfully mutated the Object referenced by the `reference_id`. The value in this field must match a `cid` previously specified by a Command by the `_writes` parameter on a prior Command for this payment Object.  For a payment Command, only zero or one `_reads` values should be specified since payments are only dependent upon at most prior version of an Object. A list with any other number of items results in a Command error.  If the list is empty this payment Command defines a new payment. If the list contains one item, then this Command updates the shared `PaymentObject`. |
-
-| _writes | list of str | Y | For a payment Command, the `_writes` field may only be a single value since a payment Command only operates upon one Object.  This field specifies the `reference_id` of the Object bring written. |
+| _writes | JSON object map | Y | For a payment command, the `_writes` field may only be a single key-value pair since a payment command only operates upon one object.  This field maps the `reference_id` of the object being written to its new version. |
```

Comment:
Legal wanted Command and Version and Object to be capitalized since they are defined values in the definitions list